PerformanceTest_Static_PressureDorsal.py

```diff
@@ -38,134 +37,52 @@
-        specified in XSENSOR output
-        """
-        subsetDat = self.fullDat.iloc[:,self.fullDat.columns.get_loc(colName):self.fullDat.columns.get_loc(colName)+12]
-        return(subsetDat)
-
-    
-
-def createAvgMat(inputName):
-    """ 
-    Reads in file, creates average matrix data, in the shape of the pressure sensor(s), to be plotted and features
-    are extracted. The result is a dataclass which can be used for further plotting
-    
-    inputName: string
-        filename of data static trial you are processing. 
+###############################################################################
+# Function List   
+def plotAvgStaticPressure(plantarMat, inputDC, FilePath, StaticPostion):
     """
-   
-        
-    # inputName = entries[0]
-    # dat = pd.read_csv(fPath+inputName, sep=',',skiprows = 1, header = 'infer')
-    dat = pd.read_csv(fPath+inputName, sep=',',header = 'infer')
-    subj = inputName.split(sep="_")[0]
-    config = inputName.split(sep="_")[1].split(sep=".")[0]
-    
-    if 'Insole' in dat.columns:
-        if  dat['Insole'][0] == 'Left':      # check to see if right insole used
-           
-            plantarSensel = dat.loc[:,'S_1_5':'S_31_5']
-    
-
-    if dat['Insole'][0] != 'Right' and dat['Insole'][0] != 'Left' :       # check to see if dorsal pad was used
-        
-        dorsalSensel = dat.loc[:,'S_1_1':'S_18_10']
-        
-    elif 'Insole.1' in dat.columns:
-        if dat['Insole.1'][0] != 'Right' and dat['Insole.1'][0] != 'Left' :
-
-            dorsalSensel = dat.loc[:,'S_1_1':'S_18_10']
-    
-    if  dat['Insole'][0] == 'Right':  # check to see if left insole used
-        
-        plantarSensel = dat.loc[:, 'S_1_2':'S_31_7'] 
-    
-    elif  'Insole.1' in dat.columns:
-        if dat['Insole.1'][0] == 'Right':  
-            
-            plantarSensel = dat.loc[:, 'S_1_2.1':'S_31_7']
-    
+    Plot the average static plantar and dorsal pressure
+    Function dependencies: need to use createTSmat for the appropriate array
+    shape for the plantar and dorsal pressure
+
+    Parameters
+    ----------
+    plantarMat : numpy array
+        DESCRIPTION.
+    inputDC : dataclass
+        Created from the function "createTSmat"
+    FilePath : str
+        file path string
+
+    Returns
+    -------
+    fig : matplotlib figure
+        Figure showing average dorsal and plantar pressure during the static trial
 
+    """
     
+    avgPlantar = np.mean(plantarMat, axis = 0)
+    avgDorsal = np.mean(inputDC.dorsalMat, axis = 0)
     
-    headers = plantarSensel.columns
-    store_r = []
-    store_c = []
-
-    for name in headers:
-        store_r.append(int(name.split(sep = "_")[1])-1)
-        store_c.append(int(name.split(sep = "_")[2].split(sep=".")[0])-1)
-    
-    con_press = np.zeros((np.max(store_r)+1,np.max(store_c)+1))
+    fig, (ax1, ax2) = plt.subplots(1,2)
+    ax1 = sns.heatmap(avgDorsal, ax = ax1, cmap="mako", vmin = 0, vmax = np.max(avgDorsal) * 2)
```

Comment:
for some subjects, using max pressure to set plot axis has diluted/washout color mapping mainly in dorsal plots. Not sure if this is a delimit issue - movement when we assume no movement, or something else. is there a better way to do this?